src/mappers/mapOp.ts

```diff
@@ -48,3 +54,29 @@ function mapMultiplyOp(context: ParseContext, node: CoffeeOp) {
     mapAny(context, node.second)
   );
 }
+
+function mapYieldOp(context: ParseContext, node: CoffeeOp) {
```

Comment:
`mapMultiplyOp` was as well, and then I looked it up and learned that typescript can infer the type based on what's actually returned. But I guess you prefer to have explicit return types anyway? Would be nice if there was a tslint rule for that, although from a quick glance I don't see one.